deploy/demo/quickstart/aad-pod-identity-quickstart.sh

```diff
@@ -0,0 +1,150 @@
+	validate_azure_resource_group $NOACCESSRESOURCEGROUPNAME $LOCATION # Creating/Validating the No Access Resource Group
+	validate_msi_account $MSINAME $ASSETSRESOURCEGROUPNAME # Creating/Validating the Managed Service Identity
+	echo "Assigning MSI ${MSINAME} Contributor role access to access resource group ${ACCESSRESOURCEGROUPNAME}..."
+	ACCESSRESOURCEGROUPID=$(az group create -l ${LOCATION//\"/} -n ${ACCESSRESOURCEGROUPNAME//\"/} --query id -o tsv)
+	ASSIGNCONTRIBUTOR=$(az role assignment create --assignee-object-id ${MSIPRINCIPALID//\"/} --scope ${ACCESSRESOURCEGROUPID//\"/} --role Contributor 2>/dev/null)
+	echo "Completed assigning MSI ${MSINAME} Contributor role access to access resource group ${ACCESSRESOURCEGROUPNAME}."
+	echo "Assigning MSI ${MSINAME} Reader role access to access resource group ${NOACCESSRESOURCEGROUPNAME}..."
+	NOACCESSRESOURCEGROUPID=$(az group create -l ${LOCATION//\"/} -n ${NOACCESSRESOURCEGROUPNAME//\"/} --query id -o tsv)
+	ASSIGNREADER=$(az role assignment create --assignee-object-id ${MSIPRINCIPALID//\"/} --scope ${NOACCESSRESOURCEGROUPID//\"/} --role Reader 2>/dev/null)
+	echo "Completed assigning MSI ${MSINAME} Contributor role access to access resource group ${ACCESSRESOURCEGROUPNAME}."
+	echo "Creating and updating ${PODIDENTITYJSONFILENAME} from template..."
+	jq -r --arg MSINAME "$MSINAME" '.items[].metadata.name |= $MSINAME' ./aadpodidentity-template.json . > ${PODIDENTITYJSONFILENAME//\"/}
+	CLIENTID=$(jq -r --arg MSICLIENTID "$MSICLIENTID" '.items[].spec.ClientID |= $MSICLIENTID' ${PODIDENTITYJSONFILENAME//\"/} . | grep ClientID)
+	sed -i "11s/.*/$CLIENTID/g" ${PODIDENTITYJSONFILENAME//\"/}
+	RESOURCEID=$(jq -r --arg MSIRESOURCEID "$MSIRESOURCEID" '.items[].spec.ResourceID |= $MSIRESOURCEID' ${PODIDENTITYJSONFILENAME//\"/} . | grep ResourceID)
+	sed -i "12s#.*#$RESOURCEID#g" ${PODIDENTITYJSONFILENAME//\"/}
+	echo "Completed creation of ${PODIDENTITYJSONFILENAME}."
+	echo "Creating and updating ${AZUREIDENTITYBINDINGJSONFILENAME} from template..."
+	jq -r --arg MSINAME "$MSINAME" '.items[].metadata.name |= $MSINAME' ./azureidentitybindings-template.json . > ${AZUREIDENTITYBINDINGJSONFILENAME//\"/}
+	AZUREIDENTITY=$(jq -r --arg MSINAME "$MSINAME" '.items[].spec.AzureIdentity |= $MSINAME' ${AZUREIDENTITYBINDINGJSONFILENAME//\"/} . | grep -E "Identity\W")
+	sed -i "12s/.*/$AZUREIDENTITY/g" ${AZUREIDENTITYBINDINGJSONFILENAME//\"/}
+	SELECTOR=$(jq -r --arg PODIDENTITYLABEL "${PODIDENTITYLABEL//\"/}" '.items[].spec.Selector |= $PODIDENTITYLABEL' ${AZUREIDENTITYBINDINGJSONFILENAME//\"/} . | grep Selector)
+	sed -i "13s/.*/$SELECTOR/g" ${AZUREIDENTITYBINDINGJSONFILENAME//\"/}
+	echo "Completed creation of ${AZUREIDENTITYBINDINGJSONFILENAME}."
+	echo "Applying all necessary Kubernetes (RBAC config) configuration files for Azure AD Pod Identity..."
+	kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment-rbac.yaml
+	sleep 30
```

Comment:
```bash
kubectl wait pod --for=condition=Ready --timeout=30s -l component=mic
kubectl wait pod --for=condition=Ready --timeout=30s -l component=nmi
```
would be more deterministic than `sleep 30`.

Reply:
I agree with that.